lib/getNextJobs.js

```diff
@@ -23,6 +45,17 @@ const getNextJobs = (workflowGraph, config) => {
         throw new Error('Must provide a PR number with "~pr" trigger');
     }
 
+    if (
```

Comment:
Would be nice to have a comment above this block of code to clarify what it's doing